index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
```

Comment:
The project does not show the header section and it's broken. Kindly see the screenshot below and fix the issues.
Design page:
![design](https://user-images.githubusercontent.com/35504149/138149760-6f205635-b2d3-4593-8b93-82eba72a1676.png)
Yours:
![error](https://user-images.githubusercontent.com/35504149/138149786-ed140c20-5ddd-4a8b-be98-ee3d36299bf7.png)